Firmware/Marlin_main.cpp

```diff
@@ -7794,8 +7796,18 @@ SERIAL_PROTOCOLPGM("\n\n");
 		iPindaC = code_value_uint8();
 	}
 	else
+	{
 		iPindaC = eeprom_read_byte(&EEPROM_Sheets_base->s[iSel].pinda_temp);
+	}
+	
+	if (code_seen('A'))
```

Comment:
Does it make sense to change this code into this?:

```
bIsActive = (eeprom_read_byte(&(EEPROM_Sheets_base->active_sheet)) == iSel);
if (code_seen('A'))
{
    bHasIsActive = true;
    bIsActive |= code_value_uint8()
}
```

And skip the `else`? If it uses less memory it would be great :)